components/infobox/commons/infobox.lua

```diff
@@ -84,7 +84,7 @@ function Infobox:build(widgets)
 	for _, widget in ipairs(widgets) do
 		assert(Class.instanceOf(widget, Widget), 'Infobox:build can only accept Widgets')
 
-		self.content:node(widget:tryMake(self.injector))
+		self.content:node(tostring(widget))
```

Comment:
technically node should be able to eat it without tostring, right?
(just curious)

Reply:
Providing a non-html table to `:node()` causes all kinds of weird shit in the html library

Reply:
#4779 might make attaching widgets to html nodes work